src/stream/src/executor/sync_kv_log_store.rs

```diff
@@ -260,6 +260,7 @@ impl<S: StateStore> SyncedKvLogStoreExecutor<S> {
             new_log_store_state(self.table_id, local_state_store, self.serde);
         initial_write_state.init(first_write_epoch).await?;
 
+        let mut pause_stream = false;
```

Comment:
May set the initial value as `barrier.is_pause_on_startup()`

Reply:
Fixed